playbook_launch_instance.yml

```diff
@@ -50,6 +50,7 @@
             iops: 3000
             delete_on_termination: yes
         wait: yes
+        wait_timeout: 300
```

Comment:
This is default behavior, so I don't see much use in adding it.

Reply:
Fixed... meant to set to 600